src/pygama/flow/data_loader.py

```diff
@@ -0,0 +1,1060 @@
+                                            wf_table = tier_table[col]
+                                            if isinstance(
+                                                wf_table["values"],
+                                                ArrayOfEqualSizedArrays,
+                                            ):
+                                                # Allocate memory for columns for all channels
+                                                if "wf_dt" not in col_dict.keys():
+                                                    col_dict["wf_t0"] = np.empty(
+                                                        table_length,
+                                                        dtype=wf_table["t0"].dtype,
+                                                    )
+                                                    col_dict["wf_dt"] = np.empty(
+                                                        table_length,
+                                                        dtype=wf_table["dt"].dtype,
+                                                    )
+                                                    col_dict["wf_values"] = np.empty(
+                                                        (
+                                                            table_length,
+                                                            wf_table[
+                                                                "values"
+                                                            ].nda.shape[1],
+                                                        ),
+                                                        dtype=wf_table["values"].dtype,
+                                                    )
+                                                col_dict["wf_t0"][tcm_idx] = wf_table[
+                                                    "t0"
+                                                ].nda
+                                                col_dict["wf_dt"][tcm_idx] = wf_table[
+                                                    "dt"
+                                                ].nda
+                                                col_dict["wf_values"][
+                                                    tcm_idx
+                                                ] = wf_table["values"].nda
+                                            else:  # wf_values is a VectorOfVectors
+                                                log.warning(
+                                                    f"Not sure how to handle waveforms with values of type {type(tier_table[col]['values'])} yet"
+                                                )
+                                        else:
+                                            log.warning(
+                                                f"Not sure how to handle column {col} of type {type(tier_table[col])} yet"
+                                            )
```

Comment:
Wouldn't be better to throw `NotImplementedError`s here? Or is the execution safe to continue?

Reply:
I was thinking it should be fine to load the other requested columns and just skip any that are vov

Reply:
I agree with @gracesong312 that it should be okay to continue with a warning. In this case we should add something like "Skipping..." to the end of the warning messages so that it's clear to the user that they won't have that data.